include/OpenSpaceToolkit/Astrodynamics/Access.hpp

```diff
@@ -10,14 +10,14 @@
 #ifndef __OpenSpaceToolkit_Astrodynamics_Access__
 #define __OpenSpaceToolkit_Astrodynamics_Access__
 
-#include <OpenSpaceToolkit/Physics/Time/Interval.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Array.hpp>
```

Comment:
I am surprised by this change: usually C++ recommends importing in reverse: from specific to generic (the opposite of Python) due to how the `#include` system works.

Reply:
I take this back: it seems that there are guidelines promoting the opposite. I can't recall exactly why I picked that style: let's me dig deeper.

Reply:
@lucas-bremond I have specified the import order in .clang-format. If we want to reverse it we can totally do that.